portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,20 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Kwame Ampadu-Boateng's Portfolio</h1>
+
+      <p>Hello! My name is Kwame Ampadu-Boateng. I a currently a rising 
```

Comment:
Nit (short for nitpick): put a line break after the start tag if the contents span multiple lines, e.g. 

```
<p>
    Hello! My name is Kwame Ampadu-Boateng. I a currently a rising
    ...
</p>
```